src/screens/HomePage/homepage.js

```diff
@@ -0,0 +1,16 @@
+/**
+ * First Page
+ */
+ import CheckBox from '../../components/FilterComponents/checkbox/CheckBox';
+ 
+ 
+ function HomePage() {
+   return (
+     <div>
```

Comment:
you can use React fragment instead of the divs if you dont add any class or prop to the div.

https://reactjs.org/docs/fragments.html#short-syntax